### ISSUE TITLE
Registration notification

### ISSUE THREAD

**Issue by Anderson-Juhasc** [bitvagas/bitvagas#6]
Currently it seems that only redirects to the home page after registration.

**Comment by Anderson-Juhasc** [comment]
Maybe use a warning cloud.

**Comment by cesardeazevedo** [comment]
Yes, i did it because i we can't  redirect to dashboard page without email verification, so i redirect to the home page, but we should to redirect to a page with a massage to the user check your email...

**Comment by Anderson-Juhasc** [comment]
yeah
